heron/healthmgr/src/java/com/twitter/heron/healthmgr/HealthManager.java

```diff
@@ -173,6 +224,18 @@ public static void main(String[] args) throws Exception {
     }
   }
 
+  private static boolean hasOption(CommandLine cmd, CliArgs argName) {
```

Comment:
The methods are utility methods related to use of enum to replace hard-coded argument names. Instead of adding `.text` everywhere, which seems annoying, I added these utility methods